client-app/src/app/services/services.component.ts

```diff
@@ -7,7 +10,22 @@ import { Component, OnInit } from '@angular/core';
 })
 export class ServicesComponent implements OnInit {
 
-  constructor() { }
+  constructor(public dialog: MatDialog) { }
+  
+  
+  
+  
+  openDialog(): void {
+    const dialogRef = this.dialog.open(ActivityFormComponent, {
+       height: '400px',
+		width: '600px',
```

Comment:
Haven't looked at spec at all to know what our options are, but if we can get away with not hard  coding the size that'd be best.

Reply:
Shoot i thought i deleted that